cloud/scaling.rst

```diff
@@ -0,0 +1,58 @@
+Scaling
+#######
+
+Citus Cloud provides self-service scaling to deal with increased load. The web interface makes it easy to either add new worker nodes or increase existing nodes' memory capacity.
```

Comment:
If it's just changing an ec2 instance type then makes sense that CPU is increases as well. I'll confirm with the Cloud team.